### ISSUE TITLE
about the training loss?

### ISSUE THREAD

**Issue by rockywind** [tusen-ai/SST#155]
Hi,  thank you for sharing the great work.
When I load the pretrained model, the training loss will get bigger and bigger.
The config is the same as the fsdv2_argo_2x.py.
Can you give me some advices?
![image](https://github.com/tusen-ai/SST/assets/14838781/3ed1aeed-903a-4919-b07a-37eda72fa89e)

**Comment by Abyssaledge** [comment]
I do not really understand your setting. Please be more specific. Why you need the pretrained model for training?

**Comment by rockywind** [comment]
Hi, the pre-trained model has been developed using one-fifth of the avaliable datasets.
